src/workflow.ts

```diff
@@ -176,6 +179,7 @@ export class WorkflowContextImpl extends DBOSContextImpl implements WorkflowCont
     this.#dbosExec = dbosExec;
     this.isTempWorkflow = DBOSExecutor.tempWorkflowName === workflowName;
     this.applicationConfig = dbosExec.config.application;
+    this.maxRecoveryAttempts = workflowConfig.maxRecoveryAttempts ? workflowConfig.maxRecoveryAttempts : 50;
```

Comment:
It'd be nice being able to configure this through the environment

Reply:
Also, should it be minimum 1?

Reply:
Also confusing, none of our other config works this way. We'll see how people use it.